packages/fuels-test-helpers/src/lib.rs

```diff
@@ -153,53 +152,18 @@ pub async fn setup_test_provider(
     let mut config = node_config.unwrap_or_else(Config::local_node);
     config.chain_conf = chain_conf;
 
-    let srv = FuelService::new_node(config).await.unwrap();
-    let address = srv.bound_address;
+    let srv = FuelService::start(config).await?;
+
+    let address = srv.bound_address();
+
     tokio::spawn(async move {
         let _own_the_handle = srv;
         let () = futures::future::pending().await;
     });
 
-    Provider::from(address)
+    Ok(Provider::from(address)
```

Comment:
We can return the error. No need to expect
```suggestion
    Provider::from(address).await
```